<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">

<HTML>

<HEAD>

   <TITLE>UMKC SBS CBB Doctoral Faculty</TITLE>

   <META NAME="GENERATOR" CONTENT="Mozilla/3.0Gold (Win95; I) [Netscape]">

</HEAD>

<BODY>



<P><!--Created by Dorin-Bogdan Borza in Nov. 1995--><A HREF="sbshome.html"><IMG SRC="gif/sbslogo.gif" ALT="[SBS Home]" HSPACE=10 HEIGHT=97 WIDTH=103 ALIGN=LEFT></A></P>



<ADDRESS>School of Biological Sciences<BR>

University of Missouri-Kansas City<BR>

103 Biological Sciences Building<BR>

5007 Rockhill Road<BR>

Kansas City, Missouri 64110</ADDRESS>

<P><IMG USEMAP="#Navbar" SRC="gif/navbar.gif" ALT="Navigation Bar" HEIGHT=29 WIDTH=438><HR></P>



<H1 ALIGN=CENTER>Doctoral Faculty<BR> Cell Biology and Biophysics</H1>



<TABLE BORDER="0" CELLSPACING=0 CELLPADDING=0>

<TR><TD><A HREF="http://sgi.bls.umkc.edu/thomas_gj/lab-index.html"><IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=11 HEIGHT=12 BORDER=0></A></TD><TD>&nbsp;<A HREF="sRec05.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#GThomas">George J. Thomas, Jr.</A>, Head</TD></TR>



<TR><TD><A HREF="http://sgi.bls.umkc.edu/chronwal/index.html"><IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=11 HEIGHT=12 BORDER=0></A></TD><TD>&nbsp;<A HREF="sRec37.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Chronwall">Bibie M. Chronwall</A></TD></TR>



<TR><TD><A HREF="http://sgi.bls.umkc.edu/acooper/index.html"><IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=11 HEIGHT=12 BORDER=0></A></TD><TD>&nbsp;<A HREF="sRec29.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Cooper">Antony A. Cooper</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec40.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Dreyfus">Lawrence A. Dreyfus</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec07.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Esser">Alfred F. Esser</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec16.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Gogol">Edward P. Gogol</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec19.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Law">Douglas J. Law</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec13.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Livingston">Brian T. Livingston</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec03.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#MMC">Marino Martinez-Carrion</A>, Dean</TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec31.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Menees">Thomas M. Menees</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec28.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#LPlamann">Lynda Plamann</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec33.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#MPlamann">Mike Plamann</A></TD></TR>



<TR><TD><BR></TD><TD>&nbsp;<A HREF="sRec06.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Read">G. Sullivan Read</A></TD></TR>



<TR><TD><A HREF="http://sgi.bls.umkc.edu/waterborg/index.html"><IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=11 HEIGHT=12 BORDER=0></A></TD><TD>&nbsp;<A HREF="sRec12.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Waterborg">Jakob H. Waterborg</A></TD></TR>



<TR><TD><A HREF="http://sgi.bls.umkc.edu/yoder/yoder.html"><IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=11 HEIGHT=12 BORDER=0></A></TD><TD>&nbsp;<A HREF="sRec09.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0></A>&nbsp;</TD><TD>&nbsp;<A HREF="#Yoder">Marilyn Yoder</A></TD></TR>

</TABLE>



<HR>

<H2><A NAME="MMC"></A>Marino Martinez-Carrion</H2>

<P><A HREF="sRec03.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Professor and Dean <BR>

Ph.D. University of California-Berkeley<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 109 BSB<BR>

Phone: (816) 235-5246 </P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>

<UL><LI>Protein Engineering
<LI>Mechanisms of enzyme action 
<LI>Molecular basis of protein/protein and protein/ligand interaction
<LI>Membrane structure and function
<LI>Protein/membrane interaction
<LI>Protein folding
</UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>

Many events essential to cell development, growth and proliferation depend on the state of protein conformation within the cell.  These conformations are influenced by intracellular components including coenzymes, metabolites, other proteins and membranes.  Our laboratory interests pertain to the characterization of those conformations of the proteins alone or when interacting with other intracellular components, either in their final functional states or in the path to acquire those final conformations.  Particularly, we focus on the characterization of molecular events governing intracellular interactions of proteins that affect their localization and fate.  This research involves the development of experimental approaches that permit the isolation and characterization of essentially transient states which are part of the mechanism of protein assembly, transport within the cell or folding.  Preparation of specific proteins' variants through creation of mutants and chimeric forms is a preferred tool to procedure altered properties in the suspected regions of interest for proper intracellular interactions.  We concentrate on mammalian proteins, which function in the cytoplasm or mitochondria as they interact with intracellular components such as chaperones, membranes or other multienzyme/multiprotein complexes.  Since malfunctions of these intracellular activities are now being identified as important to biotechnology processes and to a growing number of health problems such as Alzheimer's, emphysema, cystic fibrosis and prion diseases, the molecular understanding of events is most relevant.  Toward that end, among others, we employ powerful physical tools such as electrospin resonance, mass spectrometry and quenched-flow fast kinetics in conjunction with fluorescence and circular dichroism responses.  Our overall aim is to correlate molecular events in <I>in vitro</I> systems to those occurring within the more complex intracellular environment.

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>

This work is supported by grants from the National Institutes of Health.

<HR>



<H2><A NAME="GThomas"></A>George J. Thomas, Jr.</H2>

<P><A HREF="sRec05.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Curators' Professor and Head <BR>

Ph.D. M.I.T. <BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 403 BSB<BR>

Phone: (816) 235-5247 </P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.
<BR>
Laboratory <A HREF="http://sgi.bls.umkc.edu/thomas_gj/lab-index.html">HomePage <IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=22 HEIGHT=24 BORDER=0></A></P>

<H3>Research Areas</H3>
<UL><LI>Virus structure and assembly</LI>
<LI>Protein-nucleic acid recognition</LI>
<LI>DNA and RNA structures </LI>
<LI>Protein conformation</LI>
<LI>Development of laser Raman spectroscopy as a probe of structure</LI>
<LI>Interaction and dynamics of biomolecules and macromolecular assemblies</LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
The principal objective of our research is to understand the molecular basis of protein-nucleic acid recognition. Current studies focus on viruses, telomeric DNA complexes and gene regulatory systems. The specific aims of this work are to: (i) determine the interactions of protein and nucleic acid subgroups leading to the proper assembly and stability of viruses and chromosomal assemblies; (ii) establish the detailed structures and conformations of nucleic acid and protein molecules in these assemblies; and (iii) identify chemical and biological factors which control stability and polymorphism in macromolecular complexes. These aims are pursued by combining biochemical and molecular biological probes with state-of-the-art spectroscopic methods, including laser Raman, ultraviolet resonance Raman (UVRR), and Fourier-transform infrared (FTIR) spectroscopy. The structural interpretation of results on viruses and other nucleoprotein assemblies is aided by parallel studies of model nucleic acids, proteins and their complexes. Studies are in progress on filamentous (M13, Pfl) and icosahedral viruses (P22, PRD1, <FONT FACE="Symbol">f</FONT>6, BPMV, CCMV), phage repressor-operator complexes, and ciliate and human telomeric DNA. 

<BR>We are also interested in biomolecular dynamics which occur on time scales greater than the periods of molecular vibrations. Work in progress includes the investigation of protein folding in the tailspike protein of bacteriophage P22, the determination of nucleic acid and protein dynamics in virion assemblies and the kinetics of structural transformations which control virion architecture during morphogenesis.  

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This research is supported by grants from the National Institutes of Health and Marion Merrell Dow Foundation. 

<HR>



<H2><A NAME="Chronwall"></A>Bibie M. Chronwall</H2>

<P><A HREF="sRec37.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. University of Uppsala, Sweden<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 014 BSB <BR>

Phone: (816) 235-1868 <BR>

E-mail: <A HREF="mailto:chronwall@umkc.edu">chronwall</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A>.<BR>
Laboratory <A HREF="http://sgi.bls.umkc.edu/chronwal/index.html">HomePage <IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=22 HEIGHT=24 BORDER=0></A></P>

<H3>Research Areas</H3>
Development of the pituitary intermediate lobe and mechanisms of coincident signaling.

<P><FONT SIZE="+1"><B>Current Interests</B></FONT><BR>
D<SUB>2</SUB> receptors are involved in depression, schizophrenia and Parkinson's disease.  The GABA<SUB>B</SUB> receptor agonist baclofen is a clinically effective muscle relaxant for treatment of spasticity in spinal cord injury and multiple sclerosis.  The overall objective of my research is to determine the mechanisms by which co-expressed dopamine D<SUB>2</SUB> and GABA<SUB>B</SUB> receptors synergistically inhibit cellular functions.  We focus on suppression of calcium channel activity, which is crucial for release of neurotransmitter vesicles and for contraction of muscle cells.  The key to treating many disorders may lies in drug therapies exploiting coincident signaling by receptors.  We are testing the hypothesis that the receptors signal via separate pathways which converge to allosterically inhibit the activity and gene expression of a specific channel type.  To dissect the signaling pathways, we step by step eliminate messenger molecules by antisense oligonucleotide knock-down and analyze the impact on function by video imaging and on gene expression by immunohistochemistry and <i>in situ</i> hybridization.  Pituitary melanotropes, cell lines transfected with either or both receptors and a melanotrope tumor cell line are used as model systems. 

<BR>Other ongoing projects concern heterogenity in gene expression among melanotropes, melanotrope-glia interactions, and innervation and development of the pituitary intermediate lobe. 



<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This work is supported by grants from the National Institutes of Health, National Science Foundation and the University of Missouri Research Board.

<HR>



<H2><A NAME="Cooper"></A>Antony A. Cooper</H2>

<P><A HREF="sRec29.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Assistant Professor<BR>

Ph.D. McGill University, Montreal, Canada<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 409 SCB 

Phone: (816) 235-1553<BR>

E-mail: <A HREF="mailto:coopera@umkc.edu">coopera</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A>.<BR>

Laboratory <A HREF="http://sgi.bls.umkc.edu/acooper/index.html">HomePage <IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=22 HEIGHT=24 BORDER=0></A></P>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
My research focuses on protein sorting and membrane trafficking within the secretory pathway of the baker's yeast <I>Saccharomyces cerevisiae</I>. Many basic cellular processes have been shown to be well conserved between higher eukaryotes and yeast, an organism well-suited to genetic, molecular, cell biological and biochemical manipulation. Proteins enter the secretory pathway through the endoplasmic reticulum (ER) where they fold, are post-translationally modified and often assembled into complexes. The ER contains a quality control system to retain and degrade both soluble and membrane proteins which have entered the secretory pathway but have failed to fold or assemble correctly. This process assures that only correctly folded/assembled proteins exit the ER as delivery of a mutant protein or partially assembled receptor complex to the cell surface may have severe and deleterious effects.  Such a degradative system is thought to (i) be involved in the degradation of the CFTR protein in cystic fibrosis (ii) contribute to the regulation of cholesterol biosynthesis through controlled degradation of HMG-CoA reductase and (iii) is hijacked by viruses such as HIV-1 and cytomegalovirus as a method to avoid detection during infection. We are utilizing <I>S. cervisiae</I>  as a model system to genetically identify the proteins comprising the quality control machinery.  Once identified, these proteins will be fully characterized to define their role in this process.

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This work is supported by grants from the American Heart Association-Kansas Affiliate and the National Institutes of Health.

<HR>



<H2><A NAME="Dreyfus"></A>Lawrence A. Dreyfus</H2>

<P><A HREF="sRec40.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. University of Kansas<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 506 SCB<BR>

Phone: (816) 235-5245 <BR>

E-mail: <A HREF="mailto:dreyfusl@umkc.edu">dreyfusl</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>
Molecular biology and biochemistry bacterial toxins.

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
The current research in my laboratory involves understanding how certain bacterial toxins function as virulence factors.  In other words, how do these highly evolved signaling molecules interact with and do damage to the host.  We are particularly interested in toxins associated with diarrheal diseases since these infections account world-wide for millions of deaths and countless numbers of clinical illness annually.  This is not to mention billions of dollars in economic loss, not only as a consequence of human illness but also to global agricultural economies.  The ability of many enteric pathogens to cause serious and potentially life-threatening illness is directly linked to their production of protein and peptide toxins.  Investigating the mechanism of action of these toxins therefore provides potential avenues for rational therapeutic intervention and/or disease prevention in the case of vaccine development.  Presently, my laboratory is studying two toxins, the first being the heat-stable enterotoxin B of <I>Escherichia coli</I> (STb).  STb is a peptide toxin which, unlike other known secretory enterotoxins, does not invoke elevation of cyclic nucleotides as a second messenger signaling mechanism.  Instead, STb appears to activate a heterotrimeric G protein response which induces the secretion of serotonin in the gut with the concomitant release of arachidonic acid from target cells followed by the formation of prostaglandin E2 (PGE2).  Serotonin and PGE2 are both known mediators of intestinal secretion and it is the release and formation of these compounds that is responsible for STb-induced fluid secretion.  We are actively examining the mechanism by which STb interacts with target cells in hopes of understanding the molecular mechanism for G protein activation, the trigger for the toxin-mediated secretory cascade.  The second toxin being studied in my laboratory is the cytolethal distending toxin (CDT) of <I>E. coli</I>.  This toxin is unique in that it induces massive cellular swelling, actin reassembly and eventual cell death in eukaryotic cells.  Cell death appears to be a function of the toxin-induced cessation of cell division.  Moreover, florescence activated cell sorting (FACS) analysis indicates that CDT arrests cell division at the G2/M phase of the cell cycle.  Our current understanding of the mechanism of CDT action is fragmentary since the toxin has yet to be purified and we have no information on holotoxin structure.  We are presently attempting to determine the mechanism of the CDT-induced cell cycle block while at the same time develop reagent for analysis of holotoxin structure.      

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This work is supported by grants from the National Institutes of Health.

<HR>



<H2><A NAME="Esser"></A>Alfred F. Esser</H2>

<P><A HREF="sRec07.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Marion Merrell Dow Professor<BR>

Ph.D. Goethe University, Frankfurt, Germany<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 417 BSB<BR>

Phone: (816) 235-5316 <BR>

E-mail: <A HREF="mailto:essera@umkc.edu">essera</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>
<UL><LI>Membrane biochemistry and biophysics</LI>
<LI>Complement biochemistry and biology</LI>
<LI>Processes involved in necrotic and in apoptotic cell death</LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
Our focus is on the role of membranes in immunologic recognition processes.  Specifically, we would like to understand how the humoral immune system, that is antibody and complement, recognizes foreign pathogens and malignant cells, and kills such cells without affecting the host's own cells. Currently we are investigating the interaction of the C9, the protein essential for cell killing, with natural and model membranes. Modern analytical techniques, such as fluorescence and EPR spectroscopy and differential scanning calorimetry, are used to monitor structural transitions in the protein and the membrane.  We feel that understanding complement-mediated cell damage will require a knowledge of the mechanisms by which the water-soluble complement proteins are converted into integral membrane proteins. Knowledge gained from such studies on model systems is then used to investigate processes that are biologically more relevant. For example, we are currently studying the resistance of virulent Gram-negative bacteria to complement killing and specifically the processes that are required for transport of C9 from the bacterial outer membrane, across the periplasmic space to the inner membrane where killing occurs. Modified C9 prepared by recombinant DNA techniques and expressed in insect cells using baculovirus transfer vectors is used in such studies.

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This research is supported by grants from the National Institutes of Health. 

<HR>



<H2><A NAME="Gogol"></A>Edward P. Gogol</H2>

<P><A HREF="sRec16.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. Yale University<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 415 BSB<BR>

Phone: (816) 235-2584 <BR>

E-mail: <A HREF="mailto:gogole@umkc.edu">gogole</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>
<UL><LI>High-resolution electron microscopy and image analysis of macromolecular assemblies</LI>
<LI>Specific projects: proteasome-regulatory protein assemblies, chaperonin-substrate complexes, DNA replication machinery; vacuolar ATPase </LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
My laboratory takes a structural approach to understanding the function and control of macromolecular complexes that perform several critical biological functions.  Our primary method is high-resolution electron microscopy of isolated specimens (negatively stained and frozen-hydrated), coupled with computer analysis of the images, and biochemical determination of the state and activity of the specimen.<BR>

A major current project is examination of the complexes that control the function of proteasomes, the proteolytic complexes responsible for most intracellular protein degradation.  We have shown that one regulatory complex, PA700, binds cooperatively to the two symmetrical ends of the cylindrical proteasome, stimulating its inherent peptidase activity.  Our quantitative correlation of structural modification with enzymatic activity supports a model for the activation mechanism based on changing the accessibility of substrate to the proteolytic cavity.<BR>

A more recent project in the lab aims at understanding the steps involved in the mechanism of chaperonin-assisted protein folding, specifically by the <I>E. coli</I> chaperonin, groEL.  We have visualized structural changes in groEL induced by binding a denatured substrate protein.  These changes indicate the nature of the early events in the promotion of protein folding by chaperonin.  Three-dimensional reconstruction of the enzyme-substrate complex will be used to better understand this process.


<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This research is supported by grants from the National Institutes of Health and the University of Missouri Research Board.

<HR>



<H2><A NAME="Law"></A>Douglas J. Law</H2>

<P><A HREF="sRec19.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Assistant Professor<BR>

Ph.D. Duke University<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 210 BSB<BR>

Phone: (816) 235-2568 <BR>

E-mail: <A HREF="mailto:lawdj@umkc.edu">lawdj</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A>.</P>

<H3>Research Areas</H3>
<UL><LI>Ultrastructural organization of proteins mediating adhesion and force transmission in skeletal and cardiac muscle</LI>
<LI>Mechanical properties of normal and pathological striated muscle</LI>
<LI>Interactions between proteins linking the cytoskeleton, plasma membrane, and extracellular matrix</LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
My research concerns the relationship between macromolecular structure and cellular function, at sites of adhesion between vertebrate striated muscle cells and the extracellular matrix.  A particularly specialized site for such adhesion occurs at the ends of vertebrate striated  muscle cells, and is called the myotendinous junction (MTJ).  MTJs have a structural organization that is optimized for the transmission of muscular forces across the cell membrane, and MTJs are enriched in proteins that are known to function in adhesion in other cell systems.  Because of these specializations, MTJs are thought to be the major site at which contractile forces are transmitted from striated muscle cells to the extracellular matrix.

<BR> Consistent with their crucial role in normal muscle performance, MTJs may be major sites of cellular damage in cases of muscle injury and disease.  In one muscle disease in particular, Duchenne muscular dystrophy, the absence of the cytoskeletal protein dystrophin from MTJs may result in a defect in the association of actin filaments with the muscle cell membrane. A continuing focus of research in my laboratory is the role of dystrophin in normal cytoskeleton-membrane interactions, as well as the relative roles that other cytoskeletal proteins have regarding adhesion in muscle cells that are missing their normal complement of dystrophin. Even in normal skeletal muscle, acute injury usually occurs at or near the MTJ; a second major area of inquiry in my laboratory is the study of cellular mechanisms of muscle injury and repair, and the relative importance of intracellular and extracellular structures to the mechanical properties of whole muscles.  Methods used to address the above problems include: 1) light and electron microscopy of normal and dystrophin-deficient skeletal muscle, 2) high-resolution immunolocalization of proteins in sectioned whole muscles as well as cultured muscle cells, 3) quantitation of the mechanical properties of intact muscles loaded in tension, and 4) ultrastructural analysis of sites of injury in these samples.

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This research is supported by a grant from the American Heart Association.

<HR>



<H2><A NAME="Livingston"></A>Brian T. Livingston</H2>

<P><A HREF="sRec13.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. University of California, Santa Barbara<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 310 BSB<BR>

Phone: (816) 235-5345 <BR>

E-mail: <A HREF="mailto:livingstonbr@umkc.edu">livingstonbr</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>
Control of cell differentiation during embryonic development.

<P><FONT SIZE="+1"><B>Current Interests</B></FONT><BR>
My research uses cell and molecular biology to study the mechanisms involved in determining cell fate and controlling morphogenesis in early development. Using sea urchin embryos, we are examining what the requirements are for cells in the early embryo to differentiate into specific tissue types at the proper time and place, and to organize themselves into the proper three-dimensional shape. We are currently studying a member of the winged helix family of transcription factors, <I>Spfkh1</I>, that is expressed exclusively in the endoderm of the developing embryo at the time it is giving rise to the embryonic gut. The appearance of mRNA encoding  <I>Spfkh1</I> is abrupt, and appears to be controlled by maternal factors. As the gut is formed, endoderm cells that move away from the point of invagination stop expressing <I>Spfkh1</I>, while those that remain near the point of invagination continue to express this mRNA until gut formation is complete. Preliminary studies indicate that  <I>Spfkh1</I> is involved in controlling morphogenetic movements of the endoderm. We plan to study both the regulation of transcription of <I>Spfkh1</I>, and the role of genes that <I>Spfkh1</I> in turn regulates.   We are also involved in a multi-lab sea urchin genome project.  The goal of this project is to clone and characterize genes that reflect global changes in spatial and temporal gene expression during development. 


<P><B><FONT SIZE="+1">Research Support</FONT></B><BR>
This work is supported by grants from the National Science Foundation, the Stowers Institute for Medical Research and the University of Missouri Research Board.

<HR>



<H2><A NAME="Menees"></A>Thomas M. Menees</H2>

<P><A HREF="sRec31.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Assistant Professor<BR>

Ph.D. Yale University<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: MG-101A<BR>

Phone: (816) 235-1849 <BR>

E-mail: <A HREF="mailto:meneest@umkc.edu">meneest</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A> and <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>
Identification of host genes important for the propagation of eukaryotic retroviruslike elements using the Ty3 element of the yeast <I>Saccharomyces cerevisiae</I> as a model system, understanding the roles of these host 
genes in the Ty3 life cycle using yeast molecular genetics.

<P><FONT SIZE="+1"><B>Current Interests</B></FONT><BR>
Our work currently focuses on the roles played by cell cycle and stress proteins in the Ty3 life cycle.  Host cell cycle factors are known to play roles in the retroviral life cycle since different aspects of the retroviral life cycle are blocked in cells arrested at different points in the cell cycle.  Ty3 also does not transpose when cells are arrested at particular points in the cell cycle.  We are analyzing Ty3 intermediates present during cell cycle arrest to define points in the Ty3 life cycle that are affected by the state of the host cell cycle.  In addition, we are using genetic screens to identify cellular genes involved in the cell cycle control of Ty3.

<BR>Many cellular stress proteins are members of universally conserved protein families and act as molecular chaperones or components of proteolytic systems. Viral infection, including infection by HIV-1, stimulates the
synthesis of stress proteins. Ty3 transposition is inhibited by the cellular stress response and it appears that Ty3 viruslike particles (analogous to viral core particles) within cells are targeted for destruction by the 
stress-inducible ubiquitin-mediated proteolytic system. I am identifying roles for stress proteins in assembly and stability of Ty3 viruslike particles using genetic screens for positive and negative factors in particle formation.  The long-term goal of this project is to understand Ty3 viruslike particle morphogenesis within the cell.

<P><B><FONT SIZE="+1">Research Support</FONT></B><BR>
This research is supported by a grant from the American Cancer Society and the University of Missouri Research Board.

<HR>



<H2><A NAME="LPlamann"></A>Lynda Plamann</H2>

<P><A HREF="sRec28.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. University of Iowa<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: M3-403 MED<BR>

Phone: (816) 235-1827<BR>

E-mail: <A HREF="mailto:plamannl@umkc.edu">plamannl</A></P>

<P>Laboratory <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>

<UL><LI>Cell-cell communication during fruiting body formation and sporulation in the soil bacterium <I>Myxococcus xanthus</I>
<LI>Exploration of intracellular and intercellular signal transduction mechanisms in <I>M. xanthus</I></LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
The ability to detect and respond to signals from the environment and neighboring cells is of fundamental biological importance.  Adaptation requires detection of the environmental stimulus, processing of the signal, and production of an appropriate response.  <I>Myxococcus xanthus</I>, a rod-shaped soil bacterium, provides an attractive model system for studies of intracellular and intercellular signaling.  When <I>M. xanthus</I> cells sense that they are starving, the cells begin to construct multicellular, spore-filled fruiting bodies.  Successful fruiting body formation requires a high cell density; if too few cells are present, the cells will fail to progress through the earliest stages of fruiting body formation.  An extracellular signal, A-signal, is produced and sensed by <I>M. xanthus</I> as a means to monitor the cell density.  We are studying three genes (<I>asgA</I>, <I>asgB</I> , and <I>asgC</I>) that participate in the A-signal-generating pathway.  Molecular genetic and biochemical analyses indicate that these genes are likely to be involved in environmental sensing and regulating gene expression.  Through our studies of the <I>asg</I> genes, we hope to gain a clearer understanding of the signal transduction mechanisms and cell-cell interactions that promote the multicellular state.

<P><B><FONT SIZE="+1">Research Support</FONT></B><BR>
This work is supported by grants from the National Institutes of Health  and the Department of Defense/Army Research Office.

<HR>



<H2><A NAME="MPlamann"></A>Mike Plamann</H2>

<P><A HREF="sRec33.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Assistant Professor<BR>

Ph.D. University of Iowa<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 416 BSB<BR>

Phone: (816) 235-2593<BR>

E-mail: <A HREF="mailto:plamannm@umkc.edu">plamannm</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A>.</P>

<H3>Research Areas</H3>
<UL><LI>Microtubule-associated motors
<LI>Organelle movement
<LI>Growth polarity
<LI>Cytoskeleton</LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
Cytoplasmic dynein is the largest and most complex of the cytoplasmic microtubule-associated motors.  Cytoplasmic dynein is required for the endocytic pathway, organization of Golgi, retrograde transport of organelles in axons, and microtubule-dependent mitotic processes.  The ability of cytoplasmic dynein to mediate efficient microtubule-dependent transport of membranous organelles requires an additional multisubunit complex known as dynactin.  Using the filamentous fungus <I>Neurospora crassa</I>, we have developed a genetic screen for the isolation of mutants lacking cytoplasmic dynein activity.  These mutants, designated &quot;ropy&quot;, have curled hyphal growth and are defective in the movement of nuclei.  Approximately 1200 <I>ro</I> mutants have been isolated, defining 23 complementation groups.  <I>ro-1</I> encodes the heavy chain of cytoplasmic dynein, while <I>ro-3</I> and <I>ro-4</I> encode the dynactin subunits p150<SUP>Glued</SUP> and ARP1, respectively.  Additional <I>ro</I> genes, including <I>ro-2</I>, <I>ro-7</I>, <I>ro-10</I>, and <I>ro-11</I>, encode novel proteins.  The objectives of our research are to utilize this genetic system to: (1) identified all <I>ro</I> genes required for cytoplasmic dynein or dynactin activity; (2) determine which <I>ro</I> gene products interact to form or regulate these complexes; and (3) investigate the relationship between <I>ro</I> genes encoding novel proteins and the cytoplasmic dynein and dynactin complexes.

<P><B><FONT SIZE="+1">Research Support</FONT></B><BR>
This work is supported by grants from the National Institutes of Health and Novo Nordisk Biotech, Inc.

<HR>



<H2><A NAME="Read"></A>G. Sullivan Read</H2>

<P><A HREF="sRec06.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. Pennsylvania State University<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 210 BSB<BR>

Phone: (816) 235-2583<BR>

E-mail: <A HREF="mailto:readgs@umkc.edu">readgs</A></P>

<P>Laboratory <A HREF="gstuddir.html">Graduate Students</A>.</P>

<H3>Research Areas</H3>
<UL><LI>Control of mRNA stability in mammalian cells</LI>
<LI>Post-transcriptional regulation of gene expression in cells infected with herpes simplex virus</LI>
<LI>Site-specific mutagenesis of viral genes affecting MRNA stability.</LI></UL>

<P><FONT SIZE="+1"><B>Current Interests</B></FONT><BR>
The focus of our research is the control of mRNA stability in mammalian cells and, specifically, in cells infected with herpes simplex virus (HSV).  HSV encodes a polypeptide, the virion host shutoff (<I>vhs</I>) protein, that induces rapid turnover of both viral and cellular MRNAS in infected cells. In cells infected with HSV, mRNA levels are determined by both the rate of transcription and the rate of mRNA degradation. By regulating the rate of mRNA turnover, the <I>vhs</I> protein plays an important role in the overall scheme of gene regulation in infected cells. Our laboratory is investigating the mechanism of action of the <I>vhs</I> protein. This work involves site-specific mutagenesis of the cloned gene and testing the activities of the mutant proteins. The studies also involve utilization of an <I>in vitro</I> mRNA degradation system to study the process of <I>vhs</I>-induced mRNA turnover in biochemical detail. The work should add to the understanding of gene regulation during an HSV infection. In addition, the <I>vhs</I> function provides an attractive model system that should yield information concerning the control of mRNA stability in uninfected cells.

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This work is supported by a grant from the National Institutes of Health.

<HR>



<H2><A NAME="Waterborg"></A>Jakob H. Waterborg</H2>

<P><A HREF="sRec12.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Associate Professor<BR>

Ph.D. Nijmegen, The Netherlands<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 414 BSB<BR>

Phone: (816) 235-2591 <BR>

E-mail: <A HREF="mailto:waterborgj@umkc.edu">waterborgj</A></P>

<P>Laboratory <A HREF="pdocdir.html">Research Staff</A>.<BR>

Laboratory and Personal <A HREF="http://sgi.bls.umkc.edu/waterborg/index.html">HomePage <IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=22 HEIGHT=24 BORDER=0></A> with detailed Research descriptions and references.</P>

<H3>Research Areas</H3>
<UL><LI>Structure-function relationships in chromatin</LI>
<LI>Post-synthetic modification of histones by acetylation</LI>
<LI>Transcriptionally active chromatin structures</LI>
<LI>Regulation of expression and use of histone protein variants</LI>
<LI>Identification of elements of histone gene control and of chromatin assembly</LI>
<LI>The stability of transcribed chromatin is different in plants than it is in animals</LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
Histones, small basic proteins that package DNA into nucleosomes and chromatin, strongly repress gene expression.  Regulated and precisely localized acetylation of core histone termini, especially histones H3 and H4, will neutralize positive lysine charges and loosen chromatin so that the gene transcription can proceed.  We study histone acetylation in alfalfa, and algae by radioactive tracers, chromatography and gel electrophoresis.  Previously this identified minor histone H3 variant form, histone H3.2 in alfalfa, as a functional Replacement Histone, representative for all plants.  Gene cloning showed that it is produced continuously and deposited on transcribed genes into labile chromatin.  We observed that gene transcription from chromatin in plants leads frequently to loss of nucleosomes, in contrast to animals where transcribed chromatin is stable.  Histone H3 gene cloning has identified that high, constitutive expression of intron-bearing Replacement histone H3 genes may be facilitated by nucleosome-displacing, polypyrimidine-binding proteins.<BR>

Dynamic histone acetylation, rapid acetylation and deacetylation, in alfalfa and <i>Chlamydomonas</i>, shifted to hyperacetylation by deacetylase inhibitor Trichostatin A, is distinct for H3, H4 and H2B, is limited to the transcribed fraction of the genome, and induces hypoacetylation in alfalfa upon inhibitor destruction.<BR>

The irreversible, site-specific methylation of lysines in histone H3 is studied to determine its role in limiting the site-specificity and level of H3 acetylation in transcriptionally active chromatin.        

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>

This research is supported by a grant from the University of Missouri Research Board.

<HR></P>



<H2><A NAME="Yoder"></A>Marilyn Yoder</H2>

<P><A HREF="sRec09.html"><IMG ALT="see my slide" SRC="gif/slide.gif" WIDTH=18 HEIGHT=16 BORDER=0><FONT SIZE="1"> slide picture</FONT></A>

<BR>Assistant Professor<BR>

Ph.D. University of California-Riverside<BR>

<A HREF="faculty.html#CBB"><IMG ALT="address + phone" SRC="gif/mail.gif" WIDTH=28 HEIGHT=15 BORDER=0>

Office</A>: 010 BSB<BR>

Phone: (816) 235-1986 <BR>

E-mail: <A HREF="mailto:yoderm@umkc.edu">yoderm</A></P>

<P>Laboratory Research Staff: <A HREF="pdocdir.html">[general list]</A> or <A HREF="http://sgi.bls.umkc.edu/yoder/group.html">[short list]</A>.<BR>

Laboratory <A HREF="http://sgi.bls.umkc.edu/yoder/yoder.html">HomePage <IMG ALT="visit my homepage" SRC="gif/home.gif" WIDTH=22 HEIGHT=24 BORDER=0></A></P>

<H3>Research Areas</H3>
<UL><LI>Protein structure and function</LI>
<LI>Protein crystallization and macromolecular X-ray diffraction analysis</LI>
<LI>Protein-saccharide interactions</LI>
<LI>Secretion of proteins in gram negative bacteria</LI></UL>

<P><B><FONT SIZE="+1">Current Interests</FONT></B><BR>
My research interests are in determining the molecular interactions involved in the initiation of disease states. Currently I am looking at the role of pectate degrading enzymes in the pathogenesis of plant diseases.  Microbial plant pathogens secrete a whole battery of enzymes to attack the plant. Pectate, an acidic saccharide, is one of the most enzymatically susceptible components of the plant cell wall. Only pectate degrading enzymes have been demonstrated to be virulence factors in, and of, themselves.  They are associated with plant diseases characterized by plant tissue degradation, in particular, soft rot diseases. There are two major families of pectate degrading enzymes, the polygalacturonases (PG) and the pectate lyases (PL).  Both enzymes act on the same substrate, pectate, but differ in their enzymatic mechanism; PG's cleave pectate by hydrolysis, PL's by a &szlig;-elimination mechanism.

<BR> X-ray crystallographic analysis is the primary research tool used to understand the enzymatic properties of these enzymes. By utilizing x-ray diffraction techniques, three-dimensional models of the proteins are obtained
to atomic level resolution. The information realized from these studies will be useful in characterizing the molecular mechanism used by both classes of pectate degrading enzymes and, potentially, the rational design of protein inhibitors.

<P><FONT SIZE="+1"><B>Research Support</B></FONT><BR>
This work is supported by a grant from the National Science Foundation.



<HR>[<B><A HREF="sbshome.html">SBS Home</A></B>] [<A HREF="sRec_web.html">Slides</A>]

[<A HREF="people.html">People</A>] [<A HREF="research.html">Research

&amp; Resources</A>] [<A HREF="gradprog.html">Graduate

Prog</A>] [<A HREF="ugrdprog.html">Undergrad Prog</A>]

[<A HREF="events.html">Events</A>] [<A HREF="whatsnew.html">What's

New</A>] 

<HR></P>



<ADDRESS><B>URL:</B> http://sgi.bls.umkc.edu/doctcbb.html<BR>

<B>&copy;</B> 1995 School of Biological Sciences, UMKC.<BR>

<B>Last updated</B> on December 30, 1998 by Andrea Melton.<BR>

E-mail your <B>comments</B> to <A HREF="mailto:meltona@umkc.edu">SBS webmaster &lt;meltona@umkc.edu&gt;</A></ADDRESS>



<P><MAP NAME="Navbar"><AREA SHAPE="RECT" COORDS="0,0,44,28" ALT="SBS home" HREF="sbshome.html"><AREA SHAPE="RECT" COORDS="45,0,90,28" ALT="Slides" HREF="sRec_web.html"><AREA SHAPE="RECT" COORDS="91,0,142,28" ALT="People" HREF="people.html"><AREA SHAPE="RECT" COORDS="143,0,208,28" ALT="Research" HREF="research.html"><AREA SHAPE="RECT" COORDS="209,0,267,28" ALT="Graduate Program" HREF="gradprog.html"><AREA SHAPE="RECT" COORDS="268,0,333,28" ALT="Undergraduate Program" HREF="ugrdprog.html"><AREA SHAPE="RECT" COORDS="334,0,385,28" ALT="Events" HREF="events.html"><AREA SHAPE="RECT" COORDS="386,0,438,28" ALT="What's New" HREF="whatsnew.html"></MAP></P>





</BODY>

</HTML>



